### ISSUE TITLE
new Date().getMonth()

### ISSUE THREAD

**Issue by andcastillo** [SIB-Colombia/bambu#2]
@valegrajales  You have a problem while trying to fix the dates.  
https://github.com/SIB-Colombia/bambu/blob/master/dbscripts/data_transfer/from_postgres_to_elasticsearch/resource/index.js#L25

The function new Date().getMonth() yields 0 for January and 11 for December

**Comment by andcastillo** [comment]
A good example to check
```
> var today = new Date("1990-01-01");
> today.getDate();
31          Perhaps not what you want
> today.getUTCDate();
1
> today.getFullYear();
1989
> today.getUTCFullYear();
1990
```

**Comment by valegrajales** [comment]
Thanks for reporting the problem Andres.

Fixed in 6b9df8cc9493c632020d5c6cd56d107cced4ee75
